core/sawtooth_poet/poet_consensus/poet_fork_resolver.py

```diff
@@ -233,7 +233,8 @@ def compare_forks(self, cur_fork_head, new_fork_head):
                     cur_fork_head.header_signature[:8],
                     new_fork_head.header_signature[:8])
                 chosen_fork_head = cur_fork_head
-            else:
+            elif new_fork_head.header_signature > \
```

Comment:
Should we log a debug-level message if they are equal, for debugging purposes?

Reply:
Available logs at debug level are already sufficient to identify this condition. Do you prefer we add explicit debug message again here?

Reply:
With the other logging, is it fairly explicit? (I don't have this setup right now to see what the overall logging looks like.) If not, probably doesn't hurt to do log here as well because we are expecting this to be rare correct?

Reply:
Yes, there's a trace which says something like "resolving fork for %s block over %s block". In this case both %s will be with the same value.

It's supposed to be rare, but because of validator's defect we are seeing frequent reception of same block at consensus engine.

However, I added a trace for else case here.